### ISSUE TITLE
cant load the A-Z dataset 

### ISSUE THREAD

**Issue by nofir19942** [housecricket/How-did-I-write-an-own-OCR-program-using-Keras-and-TensorFlow-in-Python#5]
for some reason the label and the data are empty after load the CSV 
there is another way to load the CSV a-z file?
or what could be the issues ?

**Comment by housecricket** [comment]
how, can you share your results here?
